pom.xml

```diff
@@ -69,8 +69,7 @@
     <hk2.version>2.5.0</hk2.version>
     <jax-rs.version>2.0.1</jax-rs.version>
     <jsr353-ri.version>1.1.4</jsr353-ri.version>
-    <jackson.version>2.9.9</jackson.version>
-    <jackson-databind.version>2.9.9.3</jackson-databind.version>
```

Comment:
It _looks like_ this property is referenced specifically in com.datastax.oss.driver.osgi.BundleOptions... is that a problem?

Reply:
good catch, I thought I caught all the references.

Reply:
@adutra It looks like we separated the Jackson version from the Jackson databind version with this commit:
https://github.com/datastax/java-driver/commit/f6980df6e065bca4ed2db28297d56e8f55c8d514
Do you recall why? And do you think it's ok to upgrade to 2.10.0 for both?

Reply:
Yes because versions for these two artifacts started diverging a while ago.

Reply:
I think it might be easier to just keep declaring these 2 properties, even if they have the same value.